jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullAndCacheBaseImageLayerStep.java

```diff
@@ -44,10 +41,7 @@
   private final DescriptorDigest layerDigest;
   private final @Nullable Authorization pullAuthorization;
 
-  private final ListenableFuture<CachedLayer> listenableFuture;
-
   PullAndCacheBaseImageLayerStep(
-      ListeningExecutorService listeningExecutorService,
       BuildConfiguration buildConfiguration,
       ProgressEventDispatcher.Factory progressEventDispatcherFactory,
       DescriptorDigest layerDigest,
```

Comment:
Note to reviewers: notably, this step is already in the pattern of the new async redesign. This step downloads and cached the base image layer denoted by the given layer digest. No future or async business. The pull Authorization is given as a concrete value too.